### PR TITLE
Consolidate term pages into a single glossary of terms

### PR DESCRIPTION
**Pull request by elrayle**
Glossary in UI:  http://elr37-dev.library.cornell.edu/samvera_docs/glossary-2.1.html

### CODE REVIEW

**Comment by elrayle** [comment]
@julesies It would be good to have your review of this since it is an overlap with a page that you maintain.  I did not remove your page since it pertains specifically to Hyrax 2.0.  The new glossary page is specific to Hyrax 2.1.

File: pages/samvera/glossary/hyrax_2.1/glossary.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Collection
+A thematic grouping of works or collections. Any user in the system can create a Collection. Collections can be visible publicly or kept private. Collections can have a dedicated thumbnail and be shared among repository users with defined access levels: view/download or edit. Only users with access to a collection can add Works to it. Example: Professor creates a collection of Works related to her funded research project. ([more information...](collection-overview.html)) 
+
+## Collection Type
+Collection Types provide a means for configuring behavior of collections within a site.  The functionality that can be switched on/off are [nesting](collection-nesting-faq.html), [sharing](collection-sharing.html), [discovery](collection-discovery-faq.html)), multiple-membership, and branding.  Once collections of a type have been created, these configurations can no longer be changed.  A site may want to create several collection types (e.g. exhibits and user collections).  ([more information...](collection-types.html))
```
Comment (julesies) [inline pages/samvera/glossary/hyrax_2.1/glossary.md]: Suggested edit for Collection Type: 

Collection Types provide a means for defining high level need of different types of collection use cases (for example, Exhibits or Favorite Lists). Collection Types allow you to configure the behavior of collections that are enabled within a repository. The functionality that can be switched on/off are [nesting](collection-nesting-faq.html), [sharing](collection-sharing.html), [discovery](collection-discovery-faq.html)), multiple-membership, and branding.  Once collections of a type have been created, these configurations can no longer be changed.

File: pages/samvera/glossary/hyrax_2.1/glossary.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+## Collection
+A thematic grouping of works or collections. Any user in the system can create a Collection. Collections can be visible publicly or kept private. Collections can have a dedicated thumbnail and be shared among repository users with defined access levels: view/download or edit. Only users with access to a collection can add Works to it. Example: Professor creates a collection of Works related to her funded research project. ([more information...](collection-overview.html))
```
Comment (julesies) [inline pages/samvera/glossary/hyrax_2.1/glossary.md]: @erayle @chrisdaaz Git rid of or merge with below. OR what about rename to "User Collections"

File: pages/samvera/glossary/hyrax_2.1/glossary.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+## Collection Type
+Collection Types provide a means for configuring behavior of collections within a site.  The functionality that can be switched on/off are [nesting](collection-nesting-faq.html), [sharing](collection-sharing.html), [discovery](collection-discovery-faq.html)), multiple-membership, and branding.  Once collections of a type have been created, these configurations can no longer be changed.  A site may want to create several collection types (e.g. exhibits and user collections).  ([more information...](collection-types.html))
+
+## Groups
+Groups of users, controlled by administrators.  Groups can be assigned as Participants for Collection Types and Admin Sets and given Sharing roles for Collections and Works.  ([more information...](collection-sharing.html##setting-groups-vs-users-as-participants))
+
+## Mediated deposit
+The group of steps necessary for reviewing a deposited work, culminating in approval before publishing it for public viewing. Commonly used in the context of collecting Electronic Theses and Dissertations or staff-facing metadata workflows for digital collections.
```
Comment (julesies) [inline pages/samvera/glossary/hyrax_2.1/glossary.md]: Consolidate with Mediate Deposit Workflow?

File: pages/samvera/glossary/hyrax_2.1/glossary.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+A template that defines the Visibility, Release, Workflow, and Participants for an Admin Set or Collection.  The template is applied to works when the work is created.  The template is created through the UI when an Admin Set or Collection is created or edited.  Permission Templates are applied to works when the item is created.  Changes to the template are not applied to existing works and will only effect new works created after the change.
+
+**Collection Type Templates**
+A template that defines the Participants for Collection based on it's Collection Type.  The template is applied to collections when the collection is created.  The template is created through the UI when a Collection Type is created or edited.  Collection Type Templates are applied to collections when the item is created.  Changes to the template are not applied to existing collections and will only effect new collections created after the change.
+
+## Mediated Deposit Workflow
+The group of steps necessary for reviewing an ingested item and its metadata, culminating in approval before publishing it for public viewing.  ([more information...](https://github.com/samvera/sufia/wiki/Mediated-Deposit-Workflow))
```
Comment (julesies) [inline pages/samvera/glossary/hyrax_2.1/glossary.md]: Remover or consolidate with mediated deposit workflow?

File: pages/samvera/glossary/hyrax_2.1/glossary.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+A classification for content that is assigned to the Work by the depositor. Resource Types are discoverable via facet by default. Examples include: Article, Dataset, Image.
+
+## Roles
+
+**System Roles**
+
+Repository Administrator is the only system-wide role that exists out of the box. Repository Administrators:
+- Can see, edit, delete all works, collections, admin sets regardless of access controls and workflows on the objects.
+- Only Repository Administrators can
+  - create Administrative Sets
```
Comment (julesies) [inline pages/samvera/glossary/hyrax_2.1/glossary.md]: @elrayle will this be true with CE?

File: pages/samvera/glossary/hyrax_2.1/glossary.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+  - manage user roles system-wide
+  - access the repository configuration settings
+
+Registered User is any user that is logged into the repository.
+Public User is anyone visiting the public contents of the repository.
+
+**Repository Object Roles** (e.g. Collection Type, Collection, Administrative Set, and Work Roles)
```
Comment (julesies) [inline pages/samvera/glossary/hyrax_2.1/glossary.md]: Can we rename to something like:  Collection-based Roles and Permissions? And then separate and add "Work-based Roles and Permissions". Just thinking it's more repo-manager friendly/where object is a little harder to understand.

File: pages/samvera/glossary/hyrax_2.1/glossary.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+- Work Editors, Viewers ([Sharing](#sharing))
+
+**Workflow Roles**
+
+Roles that a user or group of users can perform in a workflow.  These define the functions a person performs during a step in a workflow.  ([more information on Mediated Deposit](#mediated_deposit))
+- creating work
+- approving work
+
+## Sharing
+Sharing roles for Collections allow editors of the collection to share responsibility of the collection with individual users or groups of users.  Managers are given full edit access to the collection.  Depositors can add resources to the collection.  And viewers can see the collection even when it is private.  These roles can optionally be applied to works created in the collection.  ([more information...](collection-sharing.html))
```
Comment (julesies) [inline pages/samvera/glossary/hyrax_2.1/glossary.md]: Do we need this one? Maybe implied in Roles above?

**Comment by julesies** [comment]
@elrayle I added some comments, hope they are helpful and I am happy to provide any help. If you want to ping back to specific language.

File: pages/samvera/glossary/hyrax_2.1/glossary.md (hunk 2 above)
Comment (elrayle) [reply pages/samvera/glossary/hyrax_2.1/glossary.md]: User Collection is a collection type.  There can be may types of collections.  Admin Sets are now treated in the UI as just another type of collection.  So... perhaps there could be 
```
## Collection
Description of collections in general.   Mention of collection types.  Mention of Admin Sets and User Collections as default collection types that bring in backward compatibility.
```

File: pages/samvera/glossary/hyrax_2.1/glossary.md (hunk 5 above)
Comment (elrayle) [reply pages/samvera/glossary/hyrax_2.1/glossary.md]: Users will be able to give other users permission to create Admin Sets without the other users having to be admin users.

**Comment by elrayle** [comment]
From a philosophical perspective, I am somewhat inclined to have information repeat under slightly different terms when creating a glossary.  At the very least, terms like Sharing and Participants could be in the glossary, but point the user to term Roles for more information.  

I'm thinking any term the user may run into and not be sure what it means is a good candidate for a glossary.

**Pull request merged**